xarray/plot/dataset_plot.py

```diff
@@ -245,33 +251,38 @@ def _dsplot(plotfunc):
         values are ignored.
     cmap : str or colormap, optional
         The mapping from data values to color space. Either a
-        matplotlib colormap name or object. If not provided, this will
-        be either ``viridis`` (if the function infers a sequential
-        dataset) or ``RdBu_r`` (if the function infers a diverging
-        dataset).  When `Seaborn` is installed, ``cmap`` may also be a
-        `seaborn` color palette. If ``cmap`` is seaborn color palette,
+        Matplotlib colormap name or object. If not provided, this will
+        be either ``'viridis'`` (if the function infers a sequential
+        dataset) or ``'RdBu_r'`` (if the function infers a diverging
+        dataset).
+        See the :doc:`colormap reference <matplotlib:tutorials/colors/colormaps>`
+        for options.
+
+        If *seaborn* is installed, ``cmap`` may also be a
+        `seaborn color palette <https://seaborn.pydata.org/tutorial/color_palettes.html>`_.
+        Note: if ``cmap`` is a seaborn color palette,
         ``levels`` must also be specified.
-    colors : color-like or list of color-like, optional
+    colors : str or array-like of color-like, optional
```

Comment:
From `pyplot.contour`: 
> As a shortcut, single color strings may be used in place of one-element lists, i.e. 'red' instead of ['red'] to color all levels with the same color. This shortcut does only work for color strings, not for other ways of specifying colors.

Reply:
`ds.plot.scatter(..., colors=(1, 0, 0))` does not complain, but I can't get it to change the color of the scatter marks (not even with `colors="r"`) so I might be missing something.

Reply:
Yeah, it seems like the `colors` argument is not used. I did find that you can get the marker colors to change by passing `color` or `c` with a discrete `hue_style`. However, if you have a continuous `hue_style`, passing `c` is ignored (since this is done internally), and passing `color` raises `ValueError` since it conflicts with `c`.

Reply:
I think we can address this by raising appropriate errors (or implementing it) in a future PR.

Reply:
Ok, I am interested in working on this `colors` stuff. If that is ok, I will open new issue from my comment above.

Reply:
sounds great.